### PR TITLE
[EDU-4193] refactor: update events datasets variables PT

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Updated data sources and its variables in PT.
* Updated remaining features in PT.

[EDU-4193]: https://aziontech.atlassian.net/browse/EDU-4193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by gabriel-azion — APPROVED** [review]

**Review by bru-andrade — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 1]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
-| $endpoint_type | Tipo de endpoint usado no data streaming configurado. Pode ser: `HTTP_POST`, `S3`, `ELASTICSEARCH`, `QRADAR`, `AWS_KINESIS_FIREHOSE`, `KAFKA`, `DATADOG`, `BIG_QUERY`, `SPLUNK`, `AZURE_MONITOR`, `AZURE_BLOB_STORAGE` |
-| $job_name | Identificador exclusivo da Azion para o tipo de streaming criado. Exemplo: `Data Streaming HTTP`, `Data Streaming WAF` |
-| $status_code | Código de status HTTP da requisição. Exemplo: `200` |
-| $streamed_lines | Quantidade total de linhas transmitidas para o endpoint configurado. Valor máximo de `2000`. Exemplo: `837` |
-| $time | Data e hora em que os dados foram enviados para o endpoint configurado. Exemplo: `31 October, 2023, 21:10:55` |
-| $url | URL para a qual os dados do cliente foram enviados/sink. Exemplo: `https://s3.amazonaws.com` |
+:::note
+Existem duas maneiras de acessar os logs do Real-Time Events: através da interface do [Console Azion](https://console.azion.com/) e através da API. Esta documentação fornece as variáveis exibidas na interface. Você pode usar a **API GraphQL do Real⁠-⁠Time Events** e [verificar os campos disponíveis](/pt-br/documentacao/devtools/graphql-api/recursos/campos-gql-real-time-events/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:66]:
```suggestion
Existem duas maneiras de acessar os logs do Real-Time Events: através da interface do [Azion Console](https://console.azion.com/) e através da API. Esta documentação fornece as variáveis exibidas na interface. Você pode usar a **API GraphQL do Real⁠-⁠Time Events** e [verificar os campos disponíveis](/pt-br/documentacao/devtools/graphql-api/recursos/campos-gql-real-time-events/).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 2]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
 
-A seleção de um data source no menu suspenso é obrigatória. Você pode escolher entre:
+A seleção de um data source nas abas é obrigatória. Você pode escolher entre:
 
-- [Data Streaming](/pt-br/documentacao/produtos/data-streaming/)
-- [Edge Applications](/pt-br/documentacao/produtos/edge-application/)
-- [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/)
-- [Edge Pulse](/pt-br/documentacao/produtos/edge-pulse/)
-- [WAF Events](/pt-br/documentacao/produtos/edge-firewall/web-application-firewall/)
+- HTTP Requests
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:54]: será que vale linkar com os h3?

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 3]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Remote Port | Porta remota da origem que gerou a requisição. Exemplo: `8080` |
+| Scheme | Esquema da requisição. Exemplo: `HTTP` ou `HTTPS` |
+| Server Protocol | Versão do protocolo da requisição. Exemplo: `HTTP/1.1`, `HTTP/2.0`, `HTTP/3.0` |
+| Sent HTTP Content Type | `Cabeçalho “Content-Type` enviado na resposta da origem. Exemplo: `text/html; charset=UTF-8` |
+| SSL Cipher | String da cipher utilizada para estabelecimento de conexão TLS. Exemplo: `TLS_AES_256_GCM_SHA384` |
+| SSL Protocol | Protocolo de uma conexão TLS estabelecida. Exemplo: `TLS v1.2` |
+| Stack Trace | Informa os nomes das Rules Engine da edge application ou do edge firewall executadas pela requisição. Exemplo: `{\\\"edge_firewall\\\":[\\\"Global - Definir WAF\\\"]}` |
+| Status | Código de status HTTP da requisição. Exemplo: `200` |
+| Upstream Addr | Endereço IP e porta do cliente. Também pode armazenar múltiplos servidores ou grupos de servidores. Exemplo: `192.168.1.1:80`. Quando a resposta é `127.0.0.1:1666`, o upstream é o [Azion Cells Runtime](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/). |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:99]:
```suggestion
| Upstream Addr | Endereço IP e porta do cliente. Também pode armazenar múltiplos servidores ou grupos de servidores. Exemplo: `192.168.1.1:80`. Quando a resposta é `127.0.0.1:1666`, o upstream é o [Azion Edge Runtime](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/). |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 4]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Upstream Bytes Received | Número de bytes recebidos pelo edge da origem, se o conteúdo não estiver em cache. Exemplo: `8304` |
+| Upstream Bytes Sent | Número de bytes enviados para a origem. Exemplo: `2733` |
+| Upstream Cache Status | Status do cache local do edge. Pode ser: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT` ou `-` |
+| Upstream Response Time | Tempo, em milissegundos, para o edge receber uma resposta padrão da origem, incluindo cabeçalhos e corpo. Exemplo: `0.876`. **Em caso de cache, a resposta é `-`**. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:103]:
```suggestion
| Upstream Response Time | Tempo, em milissegundos, para o edge receber uma resposta padrão da origem, incluindo cabeçalhos e corpo. Exemplo: `0.876`. **Em caso de cache, a resposta é `-`** |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 5]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Upstream Status | Código de status HTTP da origem. Se um servidor não puder ser selecionado, a variável mantém o código de status 502 (Bad Gateway). Exemplo: `200`. **Em caso de cache, a resposta é `-`**. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:104]:
```suggestion
| Upstream Status | Código de status HTTP da origem. Se um servidor não puder ser selecionado, a variável mantém o código de status 502 (Bad Gateway). Exemplo: `200`. **Em caso de cache, a resposta é `-`** |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 6]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| WAF Block | Informa se o WAF bloqueou ou não a ação. `0` quando a ação não foi bloqueada e `1` quando a ação foi bloqueada. Quando estiver em *Learning Mode*, ela não será bloqueada, independentemente do retorno. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:105]:
```suggestion
| WAF Block | Informa se o WAF bloqueou ou não a ação. `0` quando a ação não foi bloqueada e `1` quando a ação foi bloqueada. Quando estiver em *Learning Mode*, ela não será bloqueada, independentemente do retorno |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 7]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| WAF Ev Headers | Quando os cabeçalhos de solicitação enviados pelo usuário são analisados pelo módulo WAF e marcados como **bloqueados** com `$waf_block = 1`, ele contém uma string codificada em base64. Caso contrário, ele contém um caractere de traço `-`. Aplica-se aos modos WAF *Learning* ou *Blocking*. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:106]:
```suggestion
| WAF Ev Headers | Quando os cabeçalhos de requisição enviados pelo usuário são analisados pelo módulo WAF e marcados como **bloqueados** com `$waf_block = 1`, ele contém uma string codificada em base64. Caso contrário, ele contém um caractere de traço `-`. Aplica-se aos modos WAF *Learning* ou *Blocking* |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 8]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| WAF Learning | Informa se o WAF está em Learning Mode. Retorna `0` se não for e `1` se for. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:107]:
```suggestion
| WAF Learning | Informa se o WAF está em Learning Mode. Retorna `0` se não for e `1` se for |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 9]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
-| $message_content | Campo aberto com uma mensagem usada no console.log para debugging. Exemplo: `[Send event to endpoint] Generic error handler; TypeError: error sending request for url (https://http-intake.logs.datadoghq.com/v1/input): connection closed before message completed` |
-| $message_level | Mensagem com o tipo de nível para a função. Pode ser: `MDN`, `DEBUG`, `INFO`, `ERROR`, `LOG` ou `WARN` |
-| $message_source | Fonte da mensagem. Pode ser: `CONSOLE` quando as mensagens são geradas pela API do Console ou `RUNTIME` quando está relacionado a uma mensagem de erro. |
-| $time | Data e hora da requisição. Exemplo: `31 October, 2023, 21:10:55` |
-
-:::note[nota]
-Para consultar os logs de uma edge function, consulte [a documentação de Debugging](/pt-br/documentacao/produtos/edge-application/edge-functions/debugging/).
+| -------- | --------- |
+| Configuration ID | Identificador único de configuração na Azion definido no arquivo de configuração do virtual host. Exemplo: `1595368520` |
+| Edge Functions Instance ID List | Lista de functions instances que foram invocadas durante a solicitação. Exemplo: `10728` |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:126]:
```suggestion
| Edge Functions Instance ID List | Lista de functions instances que foram invocadas durante a requisição. Exemplo: `10728` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 10]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Edge Functions Initiator Type List | Lista de initiators utilizados na function, separados por`;`. Pode ser `1` (Edge Application) ou `2` (Edge Firewall). |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:127]:
```suggestion
| Edge Functions Initiator Type List | Lista de initiators utilizados na function, separados por`;`. Pode ser `1` (Edge Application) ou `2` (Edge Firewall) |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 11]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+Veja a documentação de [Debugging](/pt-br/documentacao/devtools/debugging/) para mais opções de visualização dos logs de edge functions.
 :::
 
 ---
 
-### Edge Pulse
+### Edge Functions Console
 
-Exibe os registros de eventos de requisições feitas ao seu [edge pulse](/pt-br/documentacao/produtos/edge-pulse/) utilizando os dados de desempenho medidos no navegador do usuário.
+Exibe os registros de eventos de edge applications usando o [Edge Runtime](/pt-br/documentacao/devtools/runtime/visao-geral/) retornados pelo Cells Console.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:142]:
```suggestion
Exibe os registros de eventos de edge applications usando o [Edge Runtime](/pt-br/documentacao/devtools/runtime/visao-geral/) retornados pelo Azion Console.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 12]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Remote Port | Porta remota da origem que gerou a requisição. Exemplo: `8080` |
+| Request Method | Método HTTP da requisição. Exemplo: `GET` ou `POST` |
+| Request Time | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| Request URI | URI da requisição realizada pelo usuário, sem a informação do host e de protocolo e com argumentos. Exemplo:`/v1?v=bo%20dim` |
+| Scheme | Esquema da requisição. Exemplo: `HTTP` ou `HTTPS` |
+| Solution | Identificação da edge application. Exemplo: `1321` |
+| Source | Servidor que gerou a linha do log. Exemplo: `edg-fln-ggn001p` |
+| SSL Cipher | String da cipher utilizada para estabelecimento de conexão TLS. Exemplo: `TLS_AES_256_GCM_SHA384` |
+| SSL Protocol | Protocolo de uma conexão TLS estabelecida. Exemplo: `TLS v1.2` |
+| SSL Session Reused | Retorna `r` se uma sessão SSL foi reutilizada ou`.` se não foi. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:180]:
```suggestion
| SSL Session Reused | Retorna `r` se uma sessão SSL foi reutilizada ou`.` se não foi |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 13]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Status | Código de status HTTP da requisição. Exemplo: `200` |
+| TCP Info RTT | Round-Trip Time (RTT), em milissegundos, medido pelo edge para o usuário. Disponível em sistemas que suportam a opção de socket TCP_INFO. Exemplo: `72052` |
+| Upstream Cache Status | Status do cache local do edge. Pode ser: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT` ou `-` |
+| Upstream Response Time | Tempo, em milissegundos, para o edge receber uma resposta padrão da origem, incluindo cabeçalhos e corpo. Este campo é o resultado de uma soma. Exemplo: `0.876`. **Em caso de cache, a resposta é `-`**. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:184]:
```suggestion
| Upstream Response Time | Tempo, em milissegundos, para o edge receber uma resposta padrão da origem, incluindo cabeçalhos e corpo. Este campo é o resultado de uma soma. Exemplo: `0.876`. **Em caso de cache, a resposta é `-`** |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 14]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
-| $waf_args | Os argumentos da requisição. Exemplo: `quantidade_periodos=10` |
-| $waf_learning | Informa se o WAF está em Learning Mode. Pode ser `0` ou `1` |
-| $waf_match | Lista de infrações encontradas na requisição do usuário final. É formada por elementos chave-valor: a *chave* é referente ao tipo de infração detectada; o *valor* apresenta a string que gerou a infração. Exemplo: `0:1311:BODY:ctl00_cph_jp1_dados_container_clientstate`. Saiba mais sobre as [Allowed Rules do WAF](/pt-br/documentacao/produtos/edge-firewall/web-application-firewall/#allowed-rules). |
-| $waf_score | Informa a pontuação que será incrementada em caso de match com as regras criadas para o WAF. |
-| $waf_server | Nome do host usado na solicitação WAF. Exemplo: `api-login.azion.com.br` |
-| $waf_uri | URI usado na requisição do WAF. Exemplo: `/access/v2/after-login` |
+| Bytes Sent | Número de bytes enviados para um cliente. Este campo é o resultado de uma soma. Exemplo: `191` |
+| Configuration ID | Identificador único de configuração Azion definido no arquivo de configuração do virtual host. Exemplo: `1595368520` |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:165]:
```suggestion
| Configuration ID | Identificador único de configuração na Azion definido no arquivo de configuração do virtual host. Exemplo: `1595368520` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 15]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Cache TTL | Tempo, em segundos, que o objeto em cache é considerado válido (não expirado). Após a expiração do tempo, quando ocorre uma nova requisição, o Tiered Cache consulta os dados sobre a origem (upstream). Exemplo: `31536000` |
+| Configuration ID | Identificador único de configuração na Azion definido no arquivo de configuração do virtual host. Exemplo: `1595368520` |
+| Host | Informação de host enviada na linha da requisição. Armazena: o nome do host da linha da requisição, ou o nome do host do campo Host do campo host do cabeçalho, ou o nome do servidor correspondente à requisição. Exemplo: `g1sdetynmxe0ao.map.azionedge.net` |
+| Proxy Host | Hostname sendo proxeado. Exemplo: `storage.googleapis.com:443` |
+| Proxy Status | Código de status de erro HTTP ou da origem quando nenhuma resposta é obtida da origem. Exemplo: `520`. **Em caso de cache, a resposta é `-`.** |
+| Proxy Upstream | Endereço do upstream (origem). Em alguns casos, a origem do L2 é o Image Processor (IMS) para processar a imagem e então fazer o cache. Exemplo: `ims_http` |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:204]:
```suggestion
| Proxy Upstream | Endereço do upstream (origem). Em alguns casos, a origem do Tiered Cache é o Image Processor (IMS) para processar a imagem e então fazer o cache. Exemplo: `ims_http` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 16]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Request Time | Tempo de processamento da requisição decorrido desde que os primeiros bytes foram lidos a partir do cliente, com resolução de milissegundos. Este campo é o resultado de uma soma. Exemplo: `1.19` |
+| Request URI | URI da requisição realizada pelo usuário, sem a informação do host e de protocolo e com argumentos. Exemplo: `/v1?v=bo%20dim` |
+| Scheme | Esquema da requisição. Exemplo: `HTTP` ou `HTTPS` |
+| Server Protocol | Protocolo da requisição. Exemplo: `HTTP/1.1`, `HTTP/2.0`, `HTTP/3.0` |
+| Solution | Identificação da edge application. Exemplo: `1321` |
+| Status | Código de status HTTP da requisição. Exemplo: `200` |
+| TCP Info RTT | Round-Trip Time (RTT), em milissegundos, medido pelo edge para o usuário. Disponível em sistemas que suportam a opção de socket TCP_INFO. Exemplo: `72052` |
+| Upstream Addr | Endereço IP e porta do cliente. Também pode armazenar múltiplos servidores ou grupos de servidores. Exemplo: `192.168.1.1:80`. Quando a resposta é `127.0.0.1:1666`, o upstream é o [Azion Cells Runtime](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/). |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:217]:
```suggestion
| Upstream Addr | Endereço IP e porta do cliente. Também pode armazenar múltiplos servidores ou grupos de servidores. Exemplo: `192.168.1.1:80`. Quando a resposta é `127.0.0.1:1666`, o upstream é o [Azion Cells Runtime](/pt-br/documentacao/produtos/edge-application/edge-functions/runtime/visao-geral/) |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 17]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Upstream Bytes Received | Número de bytes recebidos pelo edge da origem, se o conteúdo não estiver em cache. Exemplo: `8304` |
+| Upstream Cache Status | Status do cache local do edge. Pode ser: `MISS`, `BYPASS`, `EXPIRED`, `STALE`, `UPDATING`, `REVALIDATED`, `HIT` ou `-` |
+| Upstream Connect Time | Tempo que o edge leva para estabelecer uma conexão com a origem em milissegundos. No caso de TLS, inclui o tempo gasto no handshake. Exemplo: `0.123`. **`0` no caso do KeepAlive e `-` no caso de cache.** |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:220]:
```suggestion
| Upstream Connect Time | Tempo que o edge leva para estabelecer uma conexão com a origem em milissegundos. No caso de TLS, inclui o tempo gasto no handshake. Exemplo: `0.123`. **`0` no caso do KeepAlive e `-` no caso de cache** |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 18]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Upstream Header Time | Tempo que leva para o edge receber o cabeçalho de resposta da origem em milissegundos. Exemplo: `0.345`. **Em caso de cache, a resposta é `-`**. |
+| Upstream Response Time | Tempo, em milissegundos, para o edge receber uma resposta padrão da origem, incluindo cabeçalhos e corpo. Exemplo: `0.876`. **Em caso de cache, a resposta é `-`**. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:222]:
```suggestion
| Upstream Response Time | Tempo, em milissegundos, para o edge receber uma resposta padrão da origem, incluindo cabeçalhos e corpo. Exemplo: `0.876`. **Em caso de cache, a resposta é `-`** |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 19]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Upstream Status | Código de status HTTP da origem. Se um servidor não puder ser selecionado, a variável mantém o código de status 502 (Bad Gateway). Exemplo: `200`. **Em caso de cache, a resposta é `-`**. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:223]:
```suggestion
| Upstream Status | Código de status HTTP da origem. Se um servidor não puder ser selecionado, a variável mantém o código de status 502 (Bad Gateway). Exemplo: `200`. **Em caso de cache, a resposta é `-`** |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 20]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
 
 ---
 
-## Exibição de dados
-
-Depois de concluir os filtros e buscar resultados, seus registros aparecerão em uma tabela.
+### Activity History
 
-Cada linha representa um log diferente, o que equivale a uma ação diferente realizada pelo edge. Se você clicar em um dos logs, a linha se expandirá e fornecerá informações mais detalhadas sobre ele. As informações apresentadas variam de acordo com as especificidades de cada variável.
+Exibe os registros de eventos das atividades realizadas em uma conta no Console da Azion registradas pelo [Activity History](/pt-br/documentacao/produtos/gestao-de-contas/activity-history/). Use a [API GraphQL do Real-Time Events](/pt-br/documentacao/devtools/graphql-api/visao-geral/) para consultar até *2 anos* de logs.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:266]:
```suggestion
Exibe os registros de eventos das atividades realizadas em uma conta no Azion Console registradas pelo [Activity History](/pt-br/documentacao/produtos/gestao-de-contas/activity-history/). Use a [API GraphQL do Real-Time Events](/pt-br/documentacao/devtools/graphql-api/visao-geral/) para consultar até *2 anos* de logs.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 21]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
 
-A interface pagina por uma determinada quantidade de resultados no início, mas continuará carregando todos os resultados disponíveis à medida que você percorrer a página.
+| Variável | Descrição |
+| -------- | --------- |
+| Account ID | Identificador da conta Azion. Exemplo: `8437` |
+| Author Email | Email do usuário do Azion Console que executou a ação. Exemplo: `myemail@gmail.com` |
+| Author Name | Nome do usuário do Azion Console que executou a ação. Exemplo: `Hannah` |
+| Comment | Campo editável disponível para que usuários possam adicionar comentários ao realizarem mudanças. Exemplo: `Ação realizada durante investigação` |
+| Referer Header | Cabeçalho `Referer` da página de onde a API foi chamada. Retorna quando a chamada da API é feita a partir de uma UI. Exemplo: `Test 123` |
+| Remote Port | Porta de origem que gerou a solicitação. Exemplo: `80` |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:275]:
```suggestion
| Remote Port | Porta de origem que gerou a requisição. Exemplo: `80` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 22]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| Resource ID | Identificador único do recurso que foi criado ou modificado. Exemplo: `8190` |
+| Resource Type | Identificador do tipo de recurso que foi criado ou modificado. Exemplo: `edge_application` |
+| Request Data | Dados recebidos no payload da solicitação gerada pelo usuário. Exemplo: `{"test": 123}` |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:278]:
```suggestion
| Request Data | Dados recebidos no payload da requisição gerada pelo usuário. Exemplo: `{"test": 123}` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx [hunk 23]
```diff
@@ -41,262 +41,280 @@ Você pode usar o Real-Time Events para:
...
+| User ID | Identificador único do usuário que executou a ação. Exemplo: `999` |
+| User IP | Endereço IP do usuário/origem que gerou a requisição. Exemplo: `127.0.0.1` |
 
 ---
 
-## Atualização de dados
+## Filtro de tempo e data
 
-Depois de realizar uma consulta, você pode usar o botão **Refresh** para atualizar os dados retornados. O **Real-Time Events** repetirá a última pesquisa realizada, atualizando os dados, mas mantendo o filtro de tempo e o filtro SQL que você utilizou.
+O **Time Filter** possibilita que você refine o período para o resultado da busca de eventos. Ele é selecionado por padrão para **Last Hour**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/observe/real-time-events/real-time-events.mdx:289]:
```suggestion
O **Time Filter** possibilita que você refine o período para o resultado da busca de eventos. Ele é selecionado por padrão para **Last 15 Minutes**.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**